src/main/java/frc/robot/subsystems/climber/ClimberIOSparkMax.java

```diff
@@ -0,0 +1,88 @@
+package frc.robot.subsystems.climber;
+
+import static frc.robot.Constants.ClimberConstants.*;
+
+import com.revrobotics.CANSparkBase.ControlType;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.SparkPIDController;
+import com.revrobotics.SparkPIDController.ArbFFUnits;
+import edu.wpi.first.math.util.Units;
+import frc.robot.subsystems.climber.ClimberIO.ClimberIOInputs;
+
+/**
+ * NOTE: To use the Spark Flex / NEO Vortex, replace all instances of "CANSparkMax" with
+ * "CANSparkFlex".
+ */
+public class ClimberIOSparkMax implements ClimberIO {
+  private static final double GEAR_RATIO = 1.5;
+
+  private final CANSparkMax m_winchMotorRight = new CANSparkMax(10, MotorType.kBrushless);
+  private final CANSparkMax m_winchMotorLeft = new CANSparkMax(11, MotorType.kBrushless);
+  private final RelativeEncoder encoder = m_winchMotorRight.getEncoder();
+  private final SparkPIDController pid = m_winchMotorRight.getPIDController();
+
+  public ClimberIOSparkMax() {
+    m_winchMotorRight.restoreFactoryDefaults();
+    m_winchMotorLeft.restoreFactoryDefaults();
+
+    m_winchMotorRight.setIdleMode(CANSparkMax.IdleMode.kBrake);
+    m_winchMotorLeft.setIdleMode(CANSparkMax.IdleMode.kBrake);
+
+    m_winchMotorRight.setSmartCurrentLimit(extendCurrentLimit);
+    m_winchMotorLeft.setSmartCurrentLimit(
```

Comment:
Yep, you probably want to be able to dynamically set the current limit from a command in Climber.java. Once for zeroing and another for actual operation.